scripts/bump-holochain.sh

```diff
@@ -29,19 +29,7 @@ CURRENT_BRANCH=${1:-$(git branch --show-current)}
 SEARCH_PATTERN="no-tag"
 case $CURRENT_BRANCH in
     main)
-        SEARCH_PATTERN="^holochain-0.7.[0-9]+-dev.[0-9]+$"
-        ;;
-    main-*-rc)
-        VERSION_STR=$(echo "$CURRENT_BRANCH" | awk -F '-' '{print $2}')
-
-        # Protect against this code trying to bump 1.2.3 versions if one of those shows up.
-        SEP_COUNT=$(echo "$VERSION_STR" | tr -d -c '.' | awk '{ print length; }')
-        if [ "$SEP_COUNT" -ne "1" ]; then
-            echo "Invalid version format: $VERSION_STR"
-            exit 1
-        fi
-
-        SEARCH_PATTERN=$(echo "$VERSION_STR" | awk -F '.' 'BEGIN { OFS="" } {print "^holochain-", $1, ".", $2, ".[0-9]+-rc.[0-9]+$"}')
+        SEARCH_PATTERN="^holochain-0.[0-9]+.[0-9]+-(dev|rc).[0-9]+$"
```

Comment:
Search on anything containing `dev` or `rc`. The semantic sort should retrieve the latest. It's not a latest by date. So a 0.6.1-rc.0 shouldn't show up in an update for `main` where we have `0.7.0-dev.4` already.